### ISSUE TITLE
Insight on overrides

### ISSUE THREAD

**Issue by Jurugi** [KN4CK3R/UnrealEngineSDKGenerator#31]
Ok ,

I just wanted to say, good work on the UT4 generator. 

But compared to TheFeckless's SDK gen or traditional SDK gen, it seems like there's lots of unnecessary stuff there that makes it harder to use. 

For example, I am fine using just the dumped FVector classes. By default, replacing them to 3DVector classes just adds more work. It's way easier to instead replace FVector afterward with your class that has constructors manually instead of replace them all by default without warning.

Same with the things like adding a default namespace, and uncommenting 'all' the structs. I was more impressed with TheFeckless's generator when I realized how everything works. You can just uncomment the structs as you need them and re-arrange/use the 10 functions you need. Since you end up using 10 main funcs/structs to do all the main things, there's a lot less clutter and unnecessary code compiled in even if you compile the whole SDK.

Overall this gen is a good reference and good work, it's code and it works/ but didn't like the extra work it took figuring out the 'new features' so to speak. A lot of it is only like adding prefixes, changing things you didn't need to change, etc.. Mostly std::string and modifications there, not sure if it should be "on" by default even though it's easy to change to how you want.

tldr:  Not sure if I'm "glad that the additional things are there" or "annoyed a bit that it took longer to see what it was doing" but overall it's good.
